GPUQREngine/Source/TaskDescriptor/TaskDescriptor_flops.cpp

```diff
@@ -37,9 +37,9 @@ Int flopsFactorizeVT(int numTiles)
 // flopsFactorize
 // -----------------------------------------------------------------------------
 
-Int flopsFactorize(int m, int n)
+int64_t flopsFactorize(int m, int n)
 {
-    Int v = MIN(m, n);
+    int64_t v = MIN(m, n);
     return 2 * (m-1)                    +
            v * (6 + 4*m*n + 5*n + 2*m)  +
            ((-4*n -2*m -4) * v*(v+1)/2) +
```

Comment:
That might also have been an issue prior to this change:
Are these operations promoting the integer type to `int64_t` implicitly? ISTR that this only happens from smaller integer types to `int` (or `unsigned int`). It might be necessary to explicitly convert `m` and `n` to `int64_t` to get the expected result from these arithmetic operations if the values of `m` and `n` are very large.
Choosing an output type of `int64_t` suggests that the intention is to support large values.

Reply:
Should be handled in #343.